package Advanced.Exception.Task_3;

//import java.io.IOException;

import Advanced.Exception.Task_2.MyBestException;

/* public class Main {
    public static void main(String[] args) throws MyBestException {
    }
     try {

   } catch (MyBestException e) {
       System.out.println("ERROR");
    }


 */
